File: pset1/mario/less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Checking input
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    // Create # table
    for (int i = 1; i <= n; i++)
    {
        for (int j = n; j > i; j--)
        {
            printf(" ");
        }
        for (int k = 1; k <= i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
    return 0;
}

File: pset6/mario/less/mario.py
from cs50 import get_int

a = 0
while a < 1:
    n = get_int("Height: ")
    if n >= 1 and n <= 8:
        a += 1

i = 0
j = 0
space = n - (i + 1)

for i in range(0, n):
    for j in range(0, space):
        print(" ", end="")

    space = space - 1

    for j in range(0, i + 1):
        print("#", end="")

    print("\n", end="")
